int16[4] footForce	
int16[4] footForceEst	